client/components/Project/AddProjectComponent.vue

```diff
@@ -103,9 +103,35 @@ export default {
     showModal() {
       this.show = true;
     },
-    submit() {
-      console.log(this.fields);
-      this.hideModal();
+    async submit() {
+      const options = {
+        method: 'POST',
+        headers: {'Content-Type': 'application/json'},
+        credentials: 'same-origin',
+        body: JSON.stringify({
+          projectName: this.fields.name,
+          scheduledUpdates: [this.fields.deadline],
+          invitedUsers: this.fields.collaborators,
```

Comment:
Similar to the deadline one, this.fields.collaborators will have one empty string in it if collaborators is empty (and I think we should allow for no collaborators)

Reply:
The user in this case would just delete the entry before submitting. We will eventually have validators on the frontend too, preventing the submission of empty strings.